app/pages/HasGoodsDescriptionPage.scala

```diff
@@ -16,11 +16,20 @@
 
 package pages
 
+import models.UserAnswers
 import play.api.libs.json.JsPath
 
+import scala.util.Try
+
 case object HasGoodsDescriptionPage extends QuestionPage[Boolean] {
 
   override def path: JsPath = JsPath \ toString
 
   override def toString: String = "hasGoodsDescription"
+
```

Comment:
this was added to clean up as the description needed to be deleted if the user said they didn't have one